frontend/app/components/input-select.tsx

```diff
@@ -58,7 +58,15 @@ export function InputSelect(props: InputSelectProps) {
       >
         {options.map((optionProps, index) => {
           const inputOptionId = `${id}-select-option-${index}`;
-          return <option id={inputOptionId} data-testid={inputOptionId} key={String(optionProps.value)} {...optionProps} />;
+          return (
+            <option
+              className="disabled:pointer-events-none disabled:bg-gray-100 disabled:text-gray-500 disabled:opacity-70"
```

Comment:
Add the following css in [tailwind.css](https://github.com/DTS-STN/future-sir/blob/main/frontend/app/tailwind.css#L354-L356) instead.

```diff
  option {
    color: var(--color-black);
  }
+  option:disabled {
+    color: var(--color-gray-400);
+  }
```

Reply:
![image](https://github.com/user-attachments/assets/9f7607b5-01ab-47c4-afeb-3933284c1ef6)